src/components/CommentInfo/CommentInfo.jsx

```diff
@@ -1,3 +1,25 @@
-export const CommentInfo = () => (
-  <>Put the comment here</>
+import './CommentInfo.scss';
+
+export const CommentInfo = ({ comment }) => {
+  const { .. } = comment;
```

Comment:
looks strange, you probably want to get fields from comment here
```suggestion
  const { name, email, body } = comment;
```